### PR TITLE
Mad dev 02 04

### PR DESCRIPTION
**Pull request by mad-jsanjuan**

### CODE REVIEW

File: Host/Mdis_Test/Mdis_Test.sh [hunk 1]
```diff
@@ -320,7 +320,7 @@ function cleanOnExit() {
         # Kill process
         echo "${LogPrefix} kill process ${MdisTestBackgroundPID}"
 
-        if ! kill ${MdisTestBackgroundPID}
+        if ! kill ${MdisTestBackgroundPID} 2>/dev/null
```
Comment (mad-jsanjuan) [inline Host/Mdis_Test/Mdis_Test.sh:323]: Is this just making this error message get ignored?

File: Common/Conf.sh [hunk 2]
```diff
@@ -63,22 +63,22 @@ TEST_CASES_MAP["502"]="bl70_boxpc"
...
 MenPcPassword="men"
 
 # Address of device that will be changing status of inputs in tested device 
-MenBoxPcIpAddr="10.1.1.78"
+MenBoxPcIpAddr="11.10.10.10"
 INPUT_SWITCH_TIMEOUT=10 #seconds
 
 # Credentials, address, and command to download Git repository with Test Cases source
-GitTestSourcesBranch="jpe-dev-02_04"
+GitTestSourcesBranch="mad-dev-02_04"
```
Comment (mad-jsanjuan) [inline Common/Conf.sh]: default this to mad-dev.

File: Common/Conf.sh [hunk 3]
```diff
@@ -63,22 +63,22 @@ TEST_CASES_MAP["502"]="bl70_boxpc"
...
 GitTestSourcesCmd="git clone -b ${GitTestSourcesBranch} https://github.com/MEN-Mikro-Elektronik/mdis_test_system.git"
 
 # Credentials, address, and command to download Git repository with 13MD05-90 sources
-GitMdisBranch="jpe-dev"
+GitMdisBranch="master"
```
Comment (mad-jsanjuan) [inline Common/Conf.sh]: default this to mad-dev.

File: Host/Mdis_Test/Mdis_Test.sh (hunk 1 above)
Comment (mad-jrodriguez) [reply Host/Mdis_Test/Mdis_Test.sh:323]: Yes, this change is for only hiding the possible error logs from kill command.

**Review by mad-jrodriguez — APPROVED** [review]

**Pull request merged**